docs/sources/visualize/grafana.md

```diff
@@ -4,9 +4,8 @@ menuTitle: Visualize data
 description: Visualize your log data with Grafana
 
 aliases:
-   - /docs/loki/latest/operations/grafana/
-   - /docs/loki/latest/visualize/grafana/
-   - /docs/loki/latest/getting-started/grafana/
+   - ../operations/grafana/
```

Comment:
double checking: don't you want "getting-started" on this list?

Reply:
Thanks for checking, this file was under "getting started" way back in version 2.4, I was thinking that was a long time ago, but it looks like it wasn't even two years ago.  I'll add that back in.